src/database/service.py

```
@@ -0,0 +1,567 @@
+    def close(self) -> None:
+        """Close database connections."""
+        # DatabaseManager handles closing
+        pass
+
+    def create_user_profile(
+        self,
+        user_info,
+        birth_date: date,
+        subscription_type: SubscriptionType = SubscriptionType.BASIC,
+        notifications: bool = DEFAULT_NOTIFICATIONS_ENABLED,
+        notifications_day: WeekDay = WeekDay(DEFAULT_NOTIFICATIONS_DAY),
```

Ruff (0.13.1)

Do not perform function call `WeekDay` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)